webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/index.js

```diff
@@ -0,0 +1,23 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as actions from './PersonalAccessTokensActions';
+import PersonalAccessTokens from './PersonalAccessTokens';
+import reducer from './PersonalAccessTokensReducer';
+
+const mapStateToProps = (
+  { personalAccessTokens: { newPersonalAccessToken, tokens } },
+  ownProps
+) => ({
+  tokens,
+  newPersonalAccessToken,
+  ...ownProps,
+});
+
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+export const reducers = { personalAccessTokens: reducer };
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(PersonalAccessTokens);
```

Comment:
We added redux hooks which replace the need for  `connect` HOC, I personally find hooks much more readable and flowing.
For dispatching an action  you can use `useDispatch`
For getting data from the redux' store you can use `useSelectors`